src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Infrastructure/KestrelTrace.cs

```diff
@@ -30,6 +30,9 @@ public class KestrelTrace : IKestrelTrace
         private static readonly Action<ILogger, string, Exception> _connectionDisconnect =
             LoggerMessage.Define<string>(LogLevel.Debug, 10, @"Connection id ""{ConnectionId}"" disconnecting.");
 
+        private static readonly Action<ILogger, string, Exception> _connectionRejected =
+            LoggerMessage.Define<string>(LogLevel.Warning, 11, @"Connection id ""{ConnectionId}"" rejected.");
```

Comment:
If we reserved an ID in LibuvTrace that was already used in KestrelTrace, I would have been fine with it, but since this ID was previously used for the same logger name, let's change this. To be safe I always choose an ID above all the existing ones defined in that project.